### PR TITLE
Basic EMax Skyline32 Acro board support

### PR DESCRIPTION
**Pull request by notoriou5**
* Adding EMax skyline32 acro board
* Adding sample driver to read mpu6050 data

### CODE REVIEW

**Review by helo9 — CHANGES_REQUESTED** [review]
LaGTM ;)

The datasheet is a bit too big, I would say and found some stuff I did wrong, also present in your copy ;)

File: boards/arm/emax_skyline32/upload.jlink [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Device STM32F103CB
```
[the remark this answers is not in the document]
Comment (notoriou5) [reply boards/arm/emax_skyline32/upload.jlink]: I deleted it but did not open a PR for the CC3D yet

File: samples/drivers/mpu6050/src/main.c [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+	}
+
+    if (!device_is_ready(mpu6050)) {
+        printk("MPU6050 is not ready!");
+        return;
+    }
+
+    // This needs to be done to wake up the MPU6050. Temporary solution
+    // till the zephyr upstream driver is fixed.
+    ret = i2c_reg_write_byte(i2c_dev, (uint16_t) 0x68, 0x6B, 0x00);
```
[the remark this answers is not in the document]
Comment (notoriou5) [reply samples/drivers/mpu6050/src/main.c]: replaced :)

File: samples/drivers/mpu6050/src/main.c [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            ret = sensor_sample_fetch(mpu6050);
+
+            struct sensor_value gyro[3];
+            struct sensor_value accel[3];
+            struct sensor_value temp;
+
+            sensor_channel_get(mpu6050, SENSOR_CHAN_GYRO_XYZ, gyro);
+            sensor_channel_get(mpu6050, SENSOR_CHAN_ACCEL_XYZ, accel);
+            sensor_channel_get(mpu6050, SENSOR_CHAN_DIE_TEMP, &temp);
+
```
[the remark this answers is not in the document]
Comment (notoriou5) [reply samples/drivers/mpu6050/src/main.c:129]: didn't change anything here yet...

**Review by helo9 — APPROVED** [review]

**Pull request merged**